configs/ray-summit-stable-diffusion/aws.yaml

```diff
@@ -0,0 +1,17 @@
+cloud: education-us-west-2
+head_node:
+  instance_type: g5.4xlarge
+  resources:
+    CPU: 16
+    GPU: 1
+  flags: {}
+worker_nodes:
+  - instance_type: g5.4xlarge
+    flags: {}
+    name: '1xA10G:16CPU-64GB'
+    min_nodes: 1
+    max_nodes: 1
+    market_type: ON_DEMAND
```

Comment:
Curious why you don't want to use serverless / resource-driven scaling for this?

Also, isn't hard-coding the `cloud` here going to cause issues?

Reply:
@shomilj please let me know if this resolved 🙏

Reply:
Actually, I think this entire schema is not right -- and this won't work when you try running it end to end. Downloading the YAML from the UI and pasting it here is not workable - the YAML export follows the convention of the new SDK, and this workspace template stuff bypasses the SDK and gets sent straight to a backend representation. So you want to probably copy the structure from https://github.com/anyscale/templates/blob/main/configs/e2e-llm-workflows/aws.yaml.

And also before merging -- you should test this end to end somehow. I'm not sure how, maybe check with workspaces team.